app/src/org/commcare/android/resource/installers/FileSystemInstaller.java

```diff
@@ -71,32 +66,25 @@ public void cleanup() {
     @Override
     public boolean install(Resource r, ResourceLocation location, Reference ref, ResourceTable table, AndroidCommCarePlatform instance, boolean upgrade) throws UnresolvedResourceException, UnfullfilledRequirementsException {
         try {
-            OutputStream os;
             Reference localReference;
 
             //Moved this up before the local stuff, in case the local reference fails, we don't want to start dealing with it
-            InputStream input;
-            try {
-                input = ref.getStream();
-            } catch (FileNotFoundException e) {
+            File inputFile = new File(ref.getLocalURI());

```

Comment:
Hey,

So this won't work, since the reference here could be remote. getLocalURI only works for local files, not network files.